.github/workflows/release_script.yml

```diff
@@ -0,0 +1,24 @@
+name: Release Script
+
+on:
+    push:
+        branches:
+            - dev
+            - release
+    pull_request:
+
+jobs:
+    release_script_tests:
```

Comment:
Can you put this cute little one inside https://github.com/CanalTP/navitia/blob/dev/.github/workflows/workflow.yml ?

Reply:
and i don't know if it is possible, but can we trigger this test with the **make test** command, as the others tests ?

Reply:
Unfortunately, the release script is one folder above the rest of the code... It would be a real pain to integrate it to cmake :cry: